### DISCUSSION TITLE
How can I help?

### DISCUSSION THREAD

**Discussion by njskalski** [linuxmint/discussions#64] [Development]
Hi!

How can I help with LDME6 as a developer?

**Comment by fredcw** [comment]
Pretty much anything that contributes to cinnamon or Mint also contributes to LMDE as LMDE is just linux mint on a debian base instead of Ubuntu. Not sure if that's what you meant or if you mean LMDE issues specifically, of which I don't think there are many?

**Comment by anaximeno** [comment]
See this page: https://linuxmint-developer-guide.readthedocs.io/en/latest/
This may also help: https://github.com/orgs/linuxmint/discussions/10
